packages/components/src/Chip/Chip.css

```diff
@@ -0,0 +1,96 @@
+:root {
+  --chip-height: 40px;
+  --chip-radius: 20px;
+}
+
+.chip {
+  display: flex;
+  position: relative;
+  height: var(--chip-height);
+  padding: var(--space-small) var(--base-unit);
+  border: none;
+  border-radius: var(--chip-radius);
+  text-align: left;
+  cursor: pointer;
+  transition: all ease-in-out 0.12s;
```

Comment:
We do have some standard [timing](https://atlantis.getjobber.com/?path=/docs/design-animation--page#timing) and [easing](https://atlantis.getjobber.com/?path=/docs/design-animation--page#easing) values available

Reply:
Those 20 milliseconds really made the difference though. I have fixed this. ;)